### PR TITLE
Update create-a-phone-system-auto-attendant.md

### PR DESCRIPTION
**Pull request by waseemhashem**
@SerdarSoysal

### CODE REVIEW

**Comment by scanum** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/create-a-phone-system-auto-attendant.md [hunk 1]
```diff
@@ -216,9 +216,8 @@ If you select **Disconnect**, the caller is disconnected after the greeting play
...
 
-[!INCLUDE [preview-feature](includes/preview-feature.md)]
 
-**Dial by extension** (This preview feature is not yet available to the general public) If you enable this option, callers can connect with users in your organization by entering their phone extension. You can select which users are listed as available or not available for **Dial by extension** in the optional dial scope page. Any online user with a Phone System license, or any user hosted on-premises using Skype for Business Server, is an eligible user and can be found with Dial by extension.
+**Dial by extension** If you enable this option, callers can connect with users in your organization by entering their phone extension. You can select which users are listed as available or not available for **Dial by extension** in the optional dial scope page. Any online user with a Phone System license, or any user hosted on-premises using Skype for Business Server, is an eligible user and can be found with Dial by extension.
```
Comment (JohanFreelancer9) [inline Teams/create-a-phone-system-auto-attendant.md]:
```suggestion
**Dial by extension** If you enable this option, callers can connect with users in your organization by entering their phone extension. You can select which users are listed as available or not available for **Dial by extension** in the optional Dial Scope page. Any online user with a Phone System license, or any user hosted on-premises using Skype for Business Server, is an eligible user and can be found with Dial by extension.
```

**Comment by scanum** [comment]
@kenwith  PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by kenwith** [comment]
@scanum - I don't see that the copy edit's have been resolved yet. I believe the person who opened the PR,  @waseemhashem will need to either commit or address those copy edit suggestions. If I merge now then the copy edit will just be ignored and the original PR will be merged.

**Comment by kenwith** [comment]
@waseemhashem - would you please review the copy edit above and either commit the merge to the PR or reject?

**Comment by scanum** [comment]
@waseemhashem Following up on Ken's request. Thanks!

**Comment by kenwith** [comment]
@waseemhashem  - I am going to merge and then update the copy edit in a new PR.

**Comment by kenwith** [comment]
https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/pull/3644

**Pull request merged**